### ISSUE TITLE
Visual Studio Code: Warning: The 'no-unused' rule requires type information.

### ISSUE THREAD

**Issue by sbusch** [strax/tslint-no-unused#1]
On a fresh instance of VS Code,  installed is current version 1.24.1:

```
[Info  - 14:58:41] Linter is running.
[Info  - 14:58:42] TSLint library loaded from: /Users/sebu/Devel/qbus-js/node_modules/tslint/lib/index.js
Warning: The 'no-unused' rule requires type information.
[Info  - 14:58:44] vscode-tslint: Status is OK
```

**Comment by strax** [comment]
At the moment vscode-tslint does not support linting rules that require type information (https://github.com/Microsoft/vscode-tslint/issues/70). If you want to have support for those in the editor one solution is to use [tslint-language-service](https://github.com/angelozerr/tslint-language-service) with [this plugin](https://marketplace.visualstudio.com/items?itemName=eg2.ts-tslint) instead.

**Comment by strax** [comment]
@sbusch I took a stab at this and turns out it's possible to extract unused variable information without using the type checker. I published `0.2.0-alpha.1` which does not require type information and should work within VS Code too, so you might want to check it out and see if it works for you.
